app/uk/gov/hmrc/tariffclassificationfrontend/views/reject_case.scala.html

```diff
@@ -14,35 +14,55 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.{CaseController, RejectCaseController}
 @import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.{RejectCaseController,CaseController}
 @import uk.gov.hmrc.tariffclassificationfrontend.models.Case
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.components.input_textarea
 @import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
-@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.{contact_details, case_heading}
-@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.components.input_yes_no
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_heading
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.statuses.status_change_upload
 
-@(c: Case, form: Form[Boolean])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(c: Case, form: Form[String])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @main(bodyClasses = None, customTitle = Some(messages("page.title.reject.case"))) {
 
     @case_heading(c)
 
     <div class="grid-row">
         <div class="column-full">
-            <h2 class="heading-large mt-0">Contact the applicant and attach a copy of the email before you reject this case</h2>
+            <h2 class="heading-large mt-0">Change the status of this case to: Rejected</h2>
 
-            @contact_details(c)
+            @helper.form(action = RejectCaseController.postRejectCase(c.reference),
+            'enctype -> "multipart/form-data",
+            'id -> "reject-case") {
 
-            <h3 class="heading-medium mt-0">Have you contacted the applicant and attached a copy of the email to this case?</h3>
-
-            @helper.form(action = RejectCaseController.postRejectCase(c.reference), 'id -> "reject-case") {
                 @helper.CSRF.formField
 
-                @input_yes_no(form("state"), "reject-case_contacted-applicant")
+                <div id="add-a-note-heading">
```

Comment:
Not sure the surrounding div adds anything?

Reply:
doesn't really. Can be removed in the future